curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -6,15 +6,19 @@ dashedName: step-68
 ---
 
 # --description--
+In this step, a `meta` element will be added to the `head`. This self-closing element communicates metadata to the browser and can be written as `<meta>`. In this step, you will add a `charset` attribute which describes the character encoding to the browser.
```

Comment:
I don't think we need to tell them that we are adding a meta element to the head in the first sentence because we are going to tell them to do that in the actual instructions. Same with telling them that they are going to add a charset attribute just before the instructions tell them to add a charset attribute.